sdks/python/apache_beam/examples/complete/game/user_score.py

```diff
@@ -177,12 +178,20 @@ def format_user_score_sums(user_score):
       (user, score) = user_score
       return 'user: %s, total_score: %s' % (user, score)
 
-    (  # pylint: disable=expression-not-assigned
-        p
-        | 'ReadInputText' >> beam.io.ReadFromText(args.input)
-        | 'UserScore' >> UserScore()
-        | 'FormatUserScoreSums' >> beam.Map(format_user_score_sums)
-        | 'WriteUserScoreSums' >> beam.io.WriteToText(args.output))
+    try:
```

Comment:
I would understand changes to here and many test are not meant to be final?

If there is an exception thrown in pipeline expansion time then the pipeline is incomplete.

Reply:
The exception is thrown due to our test environments not having credentials be available. They should work in production environments. I've filed #26774 to have credentials added to the environments so the try-except blocks can be removed.

Reply:
I am hoping that this is no longer necessary, but if it were necessary, i'd expect to have reasons in the PR description. I am sure this was discussed in PR review somewhere, but comments are hard to dig.

Reply:
I'm not sure why they're there now, the issue is resolved. I'll remove them.